### PR TITLE
Fix duplicated words in the docs

### PR DESCRIPTION
**Pull request by vitaldmit**
With grep utility found some duplicated words

Automerge-Triggered-By: GH:AlexWaygood

### CODE REVIEW

File: Misc/NEWS.d/3.12.0a1.rst [hunk 1]
```diff
@@ -4559,8 +4559,7 @@ The minimum Sphinx version required to build the documentation is now 3.2.
 .. section: Documentation
 
 Augmented documentation of asyncio.create_task(). Clarified the need to keep
-strong references to tasks and added a code snippet detailing how to to
-this.
+strong references to tasks and added a code snippet detailing how to this.
```
Comment (AlexWaygood) [inline Misc/NEWS.d/3.12.0a1.rst]: May as well fix the grammar here, while we're here

```suggestion
strong references to tasks and added a code snippet detailing how to do this.
```

**Review by AlexWaygood — APPROVED** [review]
Thanks!

File: Misc/NEWS.d/3.12.0a1.rst (hunk 1 above)
Comment (vitaldmit) [reply Misc/NEWS.d/3.12.0a1.rst]: Yes of course. Sorry for my inattention

**Comment by AlexWaygood** [comment]
@vitaldmit, would you be able to create PRs backporting this to 3.11 and 3.10? If you're unfamiliar with the process, I can take care of it

**Comment by vitaldmit** [comment]
> @vitaldmit, would you be able to create PRs backporting this to 3.11 and 3.10? If you're unfamiliar with the process, I can take care of it

No, I can't able to create PRs backporting this, because I am unfamiliar with the process. Please take care of it.

**Comment by AlexWaygood** [comment]
No worries.

**Comment by AlexWaygood** [comment]
Backported to 3.11 and 3.10. Thanks @vitaldmit!

**Pull request merged**